src/vs/base/common/async.ts

```diff
@@ -238,6 +238,15 @@ export class Delayer<T> {
 		}
 	}
 
+	async fire(): TPromise<void> {
```

Comment:
Why async? Unless it returns the completionPromise.

Reply:
FYI @joaomoreno

Reply:
You're right, it shouldn't be. I used `await` and didn't remove the `async`. Will change